src/Type/InterfaceType/Uri.php

```diff
@@ -0,0 +1,19 @@
+<?php
+namespace WPGraphQL\Type\InterfaceType;
+
+class Uri {
+	public static function register_type() {
+		register_graphql_interface_type(
+			'Uri',
```

Comment:
I'm not sure how I feel about `Uri` as an interface name. Seems a little nebulous to me as to what it's trying to define. Perhaps something like `UriObject` or even `SluggableObject` / `ObjectWithUri`. I just think `Uri` sounds too much like a field name than an interface.

Reply:
Naming things is tough indeed.

It's ok to have fields and Interfaces with the same name. The `node` field and `Node` Interface are great examples of this. 

----

After researching a bit, I found that Github has an interface named `UniformResourceLocatable` with 2 fields: `resourcePath` and `url` which are of a custom `URI` Scalar type, described as: `An RFC 3986, RFC 3987, and RFC 6570 (level 4) compliant URI string.`

See below: 

<img width="1650" alt="Screen Shot 2019-12-30 at 4 36 36 PM" src="https://user-images.githubusercontent.com/1260765/71604970-ca3b7c80-2b22-11ea-92c9-01a919fd6fa8.png">

Reply:
@CodeProKid ^ In that Spirit, I think perhaps we can name it `UniformResourceIdentifier` interface?

Reply:
Update to `UniformResourceIdentifiable`